scss/_patterns_tabs.scss

```diff
@@ -71,20 +66,78 @@
         }
       }
 
+      &:active,
+      &[aria-selected='true'] {
+        // Hide the highlight when focussing (in combination with the parent
+        // states) in browsers that don't support focus-visible.
+        &:focus::before,
+        &:focus::after {
+          content: none;
+        }
+      }
+    }
+  }
+  
+  // Theming
+  @if ($theme-default-p-tabs == 'dark') {
+    .p-tabs {
+      @include vf-p-tabs-dark-theme;
+    }
+
+    .p-tabs.is-light, // deprecated, use on-light instead
+    .p-tabs.on-light {
+      @include vf-p-tabs-light-theme;
+    }
+  } @else {
+    .p-tabs {
+      @include vf-p-tabs-light-theme;
+    }
+
+    .p-tabs.is-dark, // deprecated, use on-dark instead
```

Comment:
Let's not add `is-dark` if it's deprecated already. If it's a new component with the theme, I guess we are fine with just using the new `on-dark`.

Reply:
ok, done